src/commands/bug_report.rs

```diff
@@ -0,0 +1,291 @@
+    if shell == "nu" {
+        return dirs::config_dir().map(|config_dir| config_dir.join("nushell").join("config.nu"));
+    }
+
+    home_dir().and_then(|home_dir| {
+        match shell {
+            "bash" => Some(".bashrc"),
+            "cmd" => Some("AppData/Local/clink/starship.lua"),
+            "elvish" => Some(".elvish/rc.elv"),
+            "fish" => Some(".config/fish/config.fish"),
+            "ion" => Some(".config/ion/initrc"),
+            "powershell" | "pwsh" => {
+                if cfg!(windows) {
+                    Some("Documents/PowerShell/Microsoft.PowerShell_profile.ps1")
+                } else {
+                    Some(".config/powershell/Microsoft.PowerShell_profile.ps1")
+                }
+            }
+            "tcsh" => Some(".tcshrc"),
+            "xonsh" => Some(".xonshrc"),
+            "zsh" => Some(".zshrc"),
+            _ => None,
+        }
+        .map(|path| home_dir.join(path))
+    })
+}
+
+fn get_shell_version(shell: &str) -> String {
+    let time_limit = Duration::from_millis(500);
+
+    println!("get_shell_version({shell})");
+    match shell {
+        "powershell" => exec_cmd(
+            shell,
+            &["(Get-Host | Select Version | Format-Table -HideTableHeaders | Out-String).trim()"],
+            time_limit,
+        ),
+        _ => exec_cmd(shell, &["--version"], time_limit),
+    }
+    .map_or_else(
+        || UNKNOWN_VERSION.to_string(),
+        |output| output.stdout.trim().to_string(),
+    )
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::env;
+
+    #[test]
+    fn test_make_github_link() {
+        let environment = Environment {
+            os_type: os_info::Type::Linux,
+            os_version: os_info::Version::Semantic(1, 2, 3),
+            shell_info: ShellInfo {
+                name: "test_shell".to_string(),
+                version: "2.3.4".to_string(),
+                config: "No config".to_string(),
+            },
+            terminal_info: TerminalInfo {
+                name: "test_terminal".to_string(),
+                version: "5.6.7".to_string(),
+            },
+        };
+
+        let body = get_github_issue_body(&environment);
+        let link = make_github_issue_link(&body);
+
+        assert!(link.contains(clap::crate_version!()));
+        assert!(link.contains("Linux"));
+        assert!(link.contains("1.2.3"));
+        assert!(link.contains("test_shell"));
+        assert!(link.contains("2.3.4"));
+    }
+
+    #[test]
+    #[cfg(not(windows))]
+    fn test_get_config_path() {
+        let config_path = get_config_path("bash");
+        assert_eq!(home_dir().unwrap().join(".bashrc"), config_path.unwrap());
+    }
+
+    #[test]
+    fn test_get_shell_info() {
+        let shell_info = get_shell_info();
+        assert_eq!(shell_info.name, "bash");
+        assert!(shell_info.config.contains("eval"));
+        println!("config.config: {}", shell_info.config);
+    }
```

Comment:
This test was failing so I put a `dbg!` in there to see what was going on. This printed out stuff from my bashrc. I really don't think we should be grabbing this kind of information as it could contain users ENV vars that could hold secrets

Reply:
Do you think we should make it a flag!?

Reply:
I don't we should collect this information at all. This could break trust with users if we have this in here

Reply:
I want to be clear, I like the idea of having a built in bug reporting system. I think the only things that we should collect is OS and shell. Adding in the terminal might be ok. But the user's config is definitely not ok

Reply:
I agree with @gmpinder